docs/data/registry.json

```diff
@@ -944,7 +941,10 @@
         "6.0",
         "5.0"
       ],
-      "deprecated": ["3.2"]
+      "deprecated": [
```

Comment:
:question:
Why is version 5.0 not considered deprecated on enterprise?

From the [release cycle of redis](https://redis.io/docs/about/releases/#support), I think we should consider 
- 6.0 + 6.2 (critical bugs and major security issues)
-  7.x 
as supported and the rest as EOL.
So maybe we should add 6.0 as supported?

Reply:
5.0 was deprecated, as seen in the next line.

As I [linked in the main post of the PR](https://docs.redis.com/latest/rs/installing-upgrading/product-lifecycle/#endoflife-schedule), 6.0 already reached its EOL (in May). Will add it here as well.